### DISCUSSION TITLE
Copilot is AWESOME!

### DISCUSSION THREAD

**Discussion by Bobingstern** [community/community#13547] [Copilot Conversations] [answered]
Copilot is absolutely incredible! It somehow knows exactly what i need to do and when and has been extremely useful to me when i need to write annoying functions like "get the max of an array" or "create a random sine wave". It might be nice to have an indicator telling you when copilot is "thinking" rather than just waiting for 5 seconds and nothing happening.

**Comment by davecheney** [comment, marked as answer]
> It might be nice to have an indicator telling you when copilot is "thinking" rather than just waiting for 5 seconds and nothing happening.

Check the icon in the bottom right corner; it turns into a spinner while the completion is in progress.
